h2o-algos/src/main/java/hex/tree/SharedTreeModelWithContributions.java

```diff
@@ -49,7 +49,7 @@ public Frame scoreContributions(Frame frame, Key<Frame> destination_key) {
 
   protected abstract ScoreContributionsTask getScoreContributionsTask(SharedTreeModel model);
   
-  public abstract class ScoreContributionsTask extends MRTask<ScoreContributionsTask> {
+  public class ScoreContributionsTask extends MRTask<ScoreContributionsTask> {
```

Comment:
also remove the unnecessary {} in GBMModel please

Reply:
Done.